compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -505,18 +505,34 @@ returnStatement
 
 workerInteractionStatement
     :   triggerWorker
-    |   workerReply
+    |   workerSendAsync
+    ;
+
+workerSendAsync
+    :   expression RARROW Identifier (COMMA expression)? SEMICOLON
     ;
 
 // below left Identifier is of type TYPE_MESSAGE and the right Identifier is of type WORKER or CHANNEL
 triggerWorker
-    :   expression RARROW Identifier (COMMA expression)? SEMICOLON        #invokeWorker
-    |   expression RARROW FORK SEMICOLON              #invokeFork
+    :   expression RARROW FORK SEMICOLON              #invokeFork
     ;
 
 // below left Identifier is of type WORKER or CHANNEL and the right Identifier is of type message
 workerReply
-    :   expression LARROW Identifier (COMMA expression)? SEMICOLON
+    :   LARROW Identifier (COMMA expression)?
+    ;
+
+flushWorker
+    :   FLUSH LEFT_BRACKET Identifier RIGHT_BRACKET
```

Comment:
LEFT_BRACKET and LEFT_BRACKET  are not needed. Identifier is optional.

Reply:
Fixed grammar change with 2aa0a33